workspaces/src/worker/impls.rs

```diff
@@ -129,11 +157,17 @@ where
     }
 
     /// View account details of a specific account on the network.
-    pub async fn view_account(&self, account_id: &AccountId) -> Result<AccountDetails> {
-        self.client()
-            .view_account(account_id.clone(), None)
-            .await
-            .map(Into::into)
+    pub fn view_account(&self, account_id: &AccountId) -> Query<'_, ViewAccount> {
+        Query::new(
+            self.client(),
+            ViewAccount {
+                account_id: account_id.clone(),
+            },
+        )
+    }
+
+    pub fn gas_price(&self) -> Query<'_, GasPrice> {
+        Query::new(self.client(), GasPrice)
     }
```

Comment:
curious why all methods have the `view_` prefix except this one?

Reply:
I just went with the naming here in https://docs.near.org/api/rpc/gas to stay consistent with the general naming, but since this is code, and people can lookup methods name at every keystroke with an IDE, it might be easier just to name it `view_gas_price` for that discoverability while coding. Do you have any opinions on this on which way it should lean?

Reply:
I didn't realize these method names were matching the RPC names. I think it's fine to maintain parity with those names 👍